regression/rt_gk_wham_2x2v_p1.c

```diff
@@ -110,11 +110,10 @@ struct gkyl_mirror_geo_efit_inp inp = {
 
 struct gkyl_mirror_geo_grid_inp ginp = {
   .rclose = 0.2,
-  .zmin = -2.48,
-  .zmax =  2.48,
+  .zmin = -2.0,
+  .zmax =  2.0,
```

Comment:
why? has WHAM shrunk?

Reply:
The original EFIT files did not mean to go this long. Wham has been \pm 2 meters all along